src/qt/bitcoinamountfield.cpp

```diff
@@ -179,6 +164,25 @@ class AmountSpinBox: public QAbstractSpinBox
         return QAbstractSpinBox::event(event);
     }
 
+    StepEnabled stepEnabled() const
+    {
+        StepEnabled rv = 0;
+        if (isReadOnly()) // Disable steps when AmountSpinBox is read-only
+            return StepNone;
+        if(text().isEmpty()) // Allow step-up with empty field
```

Comment:
some spaced after if nits but i assume we discussed that... but i was looking forward to once 'nit' you. :)